VagrantProvision.sh

```diff
@@ -332,13 +345,6 @@ if ! grep --quiet TOMCAT8_HOME ~/.profile; then
     source ~/.profile
 fi
 
-# Remove gdal libs installed by libgdal-dev that interfere with
-# node-export-server using gdal libs compiled from source (fgdb support)
-if [ -f "/usr/lib/libgdal.*" ]; then
-    echo "Removing GDAL libs installed by libgdal-dev..."
-    sudo rm /usr/lib/libgdal.*
-fi
-
```

Comment:
I removed the `libgdal-dev` library from the original `apt-get` call because it didn't seem to be doing anything.  It sure wasn't installing `ogrinfo`...